exampleCourse/questions/demo/annotated/engCircuit/question.html

```diff
@@ -2,22 +2,22 @@
 
 <p>Determine the {{params.ask}} of the circuit below:</p>
```

Comment:
I'd be more comfortable if the fact that a unit is required was explicit in the question.

Reply:
Will add some text clarifying this. The help text also mentions that a unit is required.

Reply:
Done here: https://github.com/PrairieLearn/PrairieLearn/pull/8179/commits/85e0c6b9be23c804969d39dee155703ecbaaa412